uswds/uswds-compile
# USWDS-Compile - Dependencies: Use Node LTS 20

**Use latest Node LTS 20 and update dependencies.** Using latest LTS version of node and updated version of USWDS and SASS. Closes #71.

## Dependencies

| Dependency    | Old     | New     |
| ------------- | ------- | ------- |
| @uswds/uswds  | 3.5.0   | 3.7.0   |
| autoprefixer  | 10.4.14 | 10.4.16 |
| sass-embedded | 1.63.6  | 1.69.5  |


## How to test

1. Go to USWDS-Sandbox repo.
2. Copy `test-uswds-compile-78-node-update` branch.
3. Run`npm install`.
4. Run tasks (there should be no regressions):
    - `npm run uswds:copyAssets`
    - `npm run watch`
